nifi-commons/nifi-xml-processing/src/main/java/org/apache/nifi/xml/processing/validation/StandardSchemaValidator.java

```diff
@@ -46,6 +47,8 @@ public void validate(final Schema schema, final Source source) {
 
         try {
             validator.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, ProcessingFeature.SECURE_PROCESSING.isEnabled());
+            validator.setProperty(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            validator.setProperty(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
```

Comment:
For consistency, the `ProcessingAttribute` enumeration includes references to these named properties, along with expected values, can you make the adjustments?

Reply:
I did try initially but SpotBugs does not seem to be smart enough to resolve the value coming from `ProcessingAttribute`. The only other option I see is to have a spotbug exclude file specifically for this and ignore the false positive for `XXE_VALIDATOR` violation. Thoughts?

Reply:
Thanks for noting that detail. Actually, instead of referencing `ProcessingFeature.SECURE_PROCESSING.isEnabled()`, creating a private static variable named `SECURE_PROCESSING_ENABLED` and setting that to `true` in this class should work. See `StandardTransformProvider` for a similar approach.

Reply:
Yep, that works, thanks!